parakh-new/include/dbmodule.php

```diff
@@ -1898,7 +1898,7 @@ function get_four_till_now_ranking_list() {
 
     /*get parakh video for login page*/
     function get_parakh_video(){
-        $video = '<video class="video" id="parakh_video" controls="" loop="" width="572" height="320"><source type="video/ogg" src="Parakh_Teaser.mp4"><source type="video/mp4" src="Parakh_Teaser.mp4"><object  type="application/x-shockwave-flash" data="Parakh_Teaser.mp4" wmode="transparent"><param name="movie" value="Parakh_Teaser.mp4"><param name="wmode" value="transparent"><param name="autostart" value="false"></object></video>';
+        $video = '<video class="video" id="parakh_video" controls="" loop="" ><source type="video/ogg" src="Parakh_Teaser.mp4"><source type="video/mp4" src="Parakh_Teaser.mp4"><object  type="application/x-shockwave-flash" data="Parakh_Teaser.mp4" wmode="transparent"><param name="movie" value="Parakh_Teaser.mp4"><param name="wmode" value="transparent"><param name="autostart" value="false"></object></video>';
```

Comment:
You don't need to call PHP for video URL. you can keep this video URL in javascript variable